server/gitlab/api.go

```diff
@@ -296,7 +296,7 @@ func (g *gitlab) GetReviews(ctx context.Context, user *UserInfo) ([]*MergeReques
 		)
 	}
 
-	var mergeRequests []*MergeRequest
+	mergeRequests := []*MergeRequest{}
```

Comment:
Confirm that this change does not break our code when we don't have any open merge requests. Similarly for `Issue`!

Reply:
checked